ts-sdk/test/client.test.ts

```diff
@@ -0,0 +1,257 @@
+import { KoraClient } from "../src/index.js";
+import {
+  createKeyPairSignerFromBytes,
+  getBase58Encoder,
+  getBase64EncodedWireTransaction,
+  getBase64Encoder,
+  getTransactionDecoder,
+  KeyPairSigner,
+  signTransaction,
+  type Transaction,
+} from "@solana/kit";
+
+function transactionFromBase64(base64: string): Transaction {
+  const encoder = getBase64Encoder();
+  const decoder = getTransactionDecoder();
+  const messageBytes = encoder.encode(base64);
+  return decoder.decode(messageBytes);
+}
+
+async function loadKeypairSignerFromEnvironmentBase58(
+  envVar: string
+): Promise<KeyPairSigner> {
+  const privateKey = process.env[envVar];
+  if (!privateKey) {
+    throw new Error(`Environment variable ${envVar} is not set`);
+  }
+  const privateKeyBytes = getBase58Encoder().encode(privateKey);
+  return createKeyPairSignerFromBytes(privateKeyBytes);
+}
+
+describe("KoraClient Integration Tests", () => {
+  let client: KoraClient;
+  let signer: KeyPairSigner;
+  const rpcUrl = process.env.KORA_RPC_URL!;
+  const testWallet = process.env.TEST_WALLET_PUBKEY!;
+  const usdcMint = process.env.USDC_MINT!;
+
+  beforeAll(async () => {
+    client = new KoraClient(rpcUrl);
+    signer = await loadKeypairSignerFromEnvironmentBase58(
+      "TEST_WALLET_PRIVATE_KEY"
```

Comment:
note: I will follow up with a script to initialize local env for tests + add integration tests to CI